packages/nextra-theme-docs/src/index.tsx

```diff
@@ -236,9 +236,11 @@ const InnerLayout = ({
 }
 
 export default function Layout(props: any): ReactElement {
-  const { route } = useRouter()
-  const context = globalThis.__nextra_pageContext__[route]
-  if (!context) throw new Error(`No content found for ${route}.`)
+  const context = usePageContext()
+  if (!context)
```

Comment:
I guess this check we can add directly to use-page-context.ts? To avoid repetition in docs and blog also it will be available for custom themes :)